source/MonoGame.Extended.Gui/Controls/Control.cs

```diff
@@ -226,7 +226,7 @@ protected TextInfo GetTextInfo(IGuiContext context, string text, Rectangle targe
             var font = Font ?? context.DefaultFont;
             var textSize = (Size)font.GetStringRectangle(text ?? string.Empty, Vector2.Zero).Size;
             var destinationRectangle = LayoutHelper.AlignRectangle(horizontalAlignment, verticalAlignment, textSize, targetRectangle);
-            var textPosition = destinationRectangle.Location.ToVector2();
```

Comment:
Point.ToVector2() is an MG extension. It doesn't exist in FNA.
An alternative is to add an extension method like Nez does.